yt_dlp/extractor/canvas.py

```diff
@@ -40,10 +41,9 @@ class CanvasIE(InfoExtractor):
     }]
     _GEO_BYPASS = False
     _HLS_ENTRY_PROTOCOLS_MAP = {
-        'HLS': 'm3u8_native',
         'HLS_AES': 'm3u8',
     }
```

Comment:
this seems to be no longer needed. Can you try changing the code to always go through `m3u8_native` and see if it works? (install pycryptodomex when testing)

Reply:
using `m3u8_native` for both HLS and HLS_AES indeed seems to give the best results: 
```
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 253
[download] Destination: Thuis - Aflevering 5090 (Seizoen 27) [pbs-pub-05a42893-a426-44a4-9971-c639c55b731f$vid-9cc1f8f8-3b16-4f3e-ac32-74b3db675c30].mp4
[download]   0.8% of ~5.80GiB at 998.05KiB/s ETA 01:40:43 (frag 2/253)
```